BlueprintUI/Sources/Element/ElementContent.swift

```diff
@@ -23,42 +23,75 @@ public struct ElementContent {
         storage = builder
     }
 
-    // MARK: Measurement & Children
+    fileprivate init(storage: ContentStorage) {
+        self.storage = storage
+    }
+
+    func measure(
+        in constraint: SizeConstraint,
+        with environment: Environment,
+        state: ElementState
+    ) -> CGSize {
+        autoreleasepool {
```

Comment:
Are these `autoreleasepool`s exclusively to avoid memory bloating? Do they cause any negative performance impact?

Reply:
Yeah, avoid big spikes. No perf impact that I can tell since it'd just be released next runloop cycle in aggregate anyway.